ansible/roles/mongo/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+- name: Install Mongodb Repo
+  copy:
+    dest: /etc/yum.repos.d/mongodb-org-7.0.repo
+    content: |
+      [mongodb-org-7.0]
+      name=MongoDB Repository
+      baseurl=https://repo.mongodb.org/yum/amazon/2023/mongodb-org/7.0/x86_64/
+      gpgcheck=1
+      enabled=1
+      gpgkey=https://pgp.mongodb.com/server-7.0.asc
```

Comment:
use file

Reply:
Ok. I have kept the contents into a file and then used the copy module.